jsonschema-core.xml

```diff
@@ -791,20 +791,22 @@
 
             <section title='Schema References With "$ref"'>
                 <t>
-                    The "$ref" keyword is used to reference a schema, and provides the ability to
-                    validate recursive structures through self-reference.
+                    The "$ref" keyword can be used to reference a schema which is to be applied to the
+                    current instance location. "$ref" is an assertion key word, which MUST produce a boolean
```

Comment:
These were your words buddy...
"Delegation means that the target of the $ref is processed against the current instance location, and the "results" (boolean assertion outcome and optionally the collected annotations) of $ref are simply the results of the target schema."
https://github.com/json-schema-org/json-schema-spec/issues/514#issue-277541274

Reply:
The part I'm objecting to is '"$ref" is an assertion keyword', which is not something that I say in that text that you quote.  I didn't explicitly classify it, and I'd be fine with not classifying it, but assertion keywords evaluate a condition based on a non-schema value (e.g. `"maxLength": 5`) which is not what `$ref` does.

Applicators apply a subschema and return its results (assertion and annotations).  `$ref` applies a subschema by reference rather than directly, so I wasn't particularly trying to fit it into the existing classifications, but if we are doing so then applicator is the correct one (and we might need to tweak the wording defining applicators if it talks specifically about an immediate subschema).

Reply:
And this change sort of obscures the purpose of $ref. We should probably still say it "provides the ability to validate recursive structures through self-reference".

Reply:
I'll chang this and mention recursive structures again.